config/schema/oe_subscriptions.settings.yml

```diff
@@ -0,0 +1,10 @@
+oe_subscriptions.settings:
+  type: config_object
+  label: 'Terms page'
+  mapping:
+    terms_url:
+      type: text
+      label: 'Page link'
```

Comment:
Since the terms and conditions are needed only for the anonymous submodule, let's move this in said module.
Add a form alter for the config form, add the field and save it in a submit.
Move this part of the schema in the submodule too.

Reply:
Moved, Added a class to handle all this as I saw for https://github.com/openeuropa/oe_showcase/pull/229
Splited tests too